### PR TITLE
Simplify statuses

### PR DESCRIPTION
**Pull request by lachmanfrantisek**
- Do not create aditional statuses.
- Run testing farm for one target after copr build-end.

----

Fixes: #330

### CODE REVIEW

File: packit_service/worker/copr_build.py [hunk 1]
```diff
@@ -396,7 +400,7 @@ def _process_failed_command(self, ex):
         else:
             output = ex.output
         msg = (
-            f"There was an error while creating a SRPM. {MSG_RETRIGGER}\n"
+            f"There was an error while creating the SRPM. {MSG_RETRIGGER}\n"
```
Comment (jpopelka) [inline packit_service/worker/copr_build.py]: I'd leave it without any article

File: packit_service/worker/testing_farm.py [hunk 2]
```diff
@@ -67,84 +69,119 @@ def __init__(
...
+            ),
+        )
 
-            stg = "-stg" if self.config.deployment == Deployment.stg else ""
-            copr_repo_name = (
-                f"packit/{self.project.namespace}-{self.project.repo}-"
-                f"{self.event.pr_id}{stg}"
+    def run_testing_farm(self, chroot: str) -> HandlerResults:
+        if chroot not in self.tests_chroots:
+            # Who triggered that?
```
Comment (jpopelka) [inline packit_service/worker/testing_farm.py]: naughty, naughty ;-)

File: packit_service/worker/testing_farm.py [hunk 3]
```diff
@@ -67,84 +69,119 @@ def __init__(
...
-            }
+        check_name = PRCheckName.get_testing_farm_check(chroot)
+        pipeline_id = str(uuid.uuid4())
+        logger.debug(f"Pipeline id: {pipeline_id}")
+        payload: dict = {
+            "pipeline": {"id": pipeline_id},
+            "api": {"token": self.config.testing_farm_secret},
+        }
+
+        logger.debug(f"Payload: {payload}")
```
Comment (jpopelka) [inline packit_service/worker/testing_farm.py]: you have one more `logger.debug(payload)` few lines below, do you need both?

File: packit_service/worker/testing_farm.py (hunk 2 above)
Comment (lachmanfrantisek) [reply packit_service/worker/testing_farm.py]: I don't want to add classical "this should not happen", but it should not really happen. I only want to be sure that we don't break anything in the future...

File: packit_service/worker/testing_farm.py (hunk 3 above)
Comment (lachmanfrantisek) [reply packit_service/worker/testing_farm.py]: Hmm, I only moved that code..;) Do we want to debug testing farm token?

Comment (jpopelka) [reply packit_service/worker/testing_farm.py]: I don't mind leaking tokens into logs. Whoever can get to the pods logs, can most likely get to secrets too anyway.

I don't understand your note about moving code. (friday, you know)

Comment (lachmanfrantisek) [reply packit_service/worker/testing_farm.py]: I will remove the payload one.

> moved code

(I haven't written those lines. I've only moved that code from other file, so I am not sure about the reason. Maybe some debugging by Rado from the early stages of testing-farm suppport?)

**Pull request merged**